meshmode/distributed.py

```diff
@@ -188,10 +190,6 @@ def make_remote_group_infos(
 @dataclass(init=True, frozen=True)
 class InterRankBoundaryInfo:
     """
-    .. attribute:: local_btag
-
-        A boundary tag for the local boundary towards the remote partition.
-
     .. attribute:: local_part_id
 
         An opaque, hashable, picklable identifier for the local partition.
```

Comment:
```suggestion
        An opaque, hashable, picklable identifier for the local part.
```
(also a few lines below) (Sorry, I know this is cleaning up my mess.)